cmd/determine-rebottle-runners.rb

```diff
@@ -50,10 +50,8 @@ def determine_rebottle_runners
         else
           ephemeral_suffix = "-#{ENV.fetch("GITHUB_RUN_ID")}"
           macos_runners = [{ runner: "#{macos_version}#{ephemeral_suffix}" }]
-          if macos_version >= :ventura
+          if macos_version >= :monterey
             macos_runners << { runner: "#{macos_version}-arm64#{ephemeral_suffix}" }
```

Comment:
If getting rid of the Big Sur check then we might as well do:

```suggestion
          macos_runners << { runner: "#{macos_version}-arm64#{ephemeral_suffix}" }
```